core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -74,7 +71,7 @@
   private final int initTimeout;
   private final int timeout;
   private final Seq<Broker> brokerSeq;
-  private final ZkClient zkClient;
+  private final ZkUtils zkUtils;
```

Comment:
@fpj Can you point me to the relevant equivalent code in Kafka? I was mostly using the KAFKA-2639 patch as guidance to get things in schema-registry and kafka-rest building cleanly again so the build pipeline in Jenkins isn't a mess. I'm actually not that familiar with the security side of ZK, so having you're eyes on these patches is very helpful.